packages/data-context/src/DataContext.ts

```diff
@@ -401,12 +395,7 @@ export class DataContext {
    * Resets all of the state for the data context,
    * so we can initialize fresh for each E2E test
    */
-  async resetForTest (modeOptions: Partial<AllModeOptions> = {}) {
-    this.debug('DataContext resetForTest')
-    if (!process.env.CYPRESS_INTERNAL_E2E_TESTING_SELF) {
-      throw new Error(`DataContext.reset is only meant to be called in E2E testing mode, there's no good use for it outside of that`)
-    }
-
+  async reinitializeCypress (modeOptions: Partial<AllModeOptions> = {}) {
```

Comment:
Does this have the same caveat as the error thrown above? Shouldn't really be thrown outside of E2E testing mode?

Reply:
This was changed to be used in the "reinitializeCypress" mutation which is called when there's an error in the launchpad and we want to reset the state of project from the top.